cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/SecurityPrivilegesTest.scala

```diff
@@ -58,82 +61,88 @@ class SecurityPrivilegesTest extends DocumentingTest with QueryStatisticsTestSup
       // image source: https://docs.google.com/drawings/d/1dueKAcaQORul-_Ocb5jK9bUkWgtQfdLdFw4uo7PFjTs/edit
     }
     section("Listing privileges", "administration-security-subgraph-show", "enterprise-edition") {
-      p("Available privileges for all roles can be seen using `SHOW PRIVILEGES`.")
-      p("include::show-all-privileges-syntax.asciidoc[]")
-      query("SHOW PRIVILEGES", assertPrivilegeShown(Seq(
-        Map("access" -> "GRANTED", "action" -> "access", "role" -> "regularUsers"),
-        Map("access" -> "DENIED", "action" -> "access", "role" -> "noAccessUsers")
-      ))) {
-        p(
-          """Lists all privileges for all roles.
-            |The table contains columns describing the privilege:
-            |
-            |* access: whether the privilege is granted or denied (whitelist or blacklist)
-            |* action: which type of privilege this is: traverse, read, match, write, a database privilege, a dbms privilege or admin
-            |* resource: what type of scope this privilege applies to: the entire dbms, a database, a graph or sub-graph access
-            |* graph: the specific database or graph this privilege applies to
-            |* segment: for sub-graph access control, this describes the scope in terms of labels or relationship types
-            |* role: the role the privilege is granted to
-            |""".stripMargin)
-        resultTable()
-      }
+      p("Available privileges for can be seen using the different `SHOW PRIVILEGES` commands.")
+      p("include::show-privileges-syntax.asciidoc[]")
+      section("Examples for listing all privileges", "administration-security-subgraph-show-all") {
+        p("Available privileges for all roles can be seen using `SHOW PRIVILEGES`.")
+        p("include::show-all-privileges-syntax.asciidoc[]")
+        query("SHOW PRIVILEGES", assertPrivilegeShown(Seq(
+          Map("access" -> "GRANTED", "action" -> "access", "role" -> "regularUsers"),
+          Map("access" -> "DENIED", "action" -> "access", "role" -> "noAccessUsers")
+        ))) {
+          p(
+            """Lists all privileges for all roles.
+              |The table contains columns describing the privilege:
+              |
+              |* access: whether the privilege is granted or denied (whitelist or blacklist)
```

Comment:
```suggestion
              |* *access*: whether the privilege is granted or denied 
```
Removing "whitelist or blacklist" since we're trying to steer away from using them. But if you feel that it is vital to keep, then please do.

Reply:
The only thing I've done on these lines is indenting since I wrapped them in subsections, I have no problem removing the `whitelist or blacklist`

Why do you wrap `access` in `*`? To make it bold font? Not sure what we have in other such places but I would guess ` instead of * (or nothing)

Reply:
To make it bold. But `would also work, in all rows.

Reply:
I think that at least the show index column description we had ` instead so I think we should keep that, I can do a new suggestion and update

Reply:
Or you have already started :P

Reply:
I was going to  🤪. I'll do it now!

Reply:
I just made one multi-line suggestion and updated them, felt faster 🤪

Reply:
So much better 👍